test/NameBinding/name_lookup.swift

```diff
@@ -292,9 +292,9 @@ class ThisDerived1 : ThisBase1 {
     super.baseInstanceVar = 42 // expected-error {{member 'baseInstanceVar' cannot be used on type 'ThisBase1'}}
     super.baseProp = 42 // expected-error {{member 'baseProp' cannot be used on type 'ThisBase1'}}
     super.baseFunc0() // expected-error {{instance member 'baseFunc0' cannot be used on type 'ThisBase1'}}
-    super.baseFunc0(ThisBase1())()
+    super.baseFunc0(ThisBase1())() // expected-error {{'super' instance member 'baseFunc0()' cannot be used on type 'ThisBase1'}}
     super.baseFunc1(42) // expected-error {{instance member 'baseFunc1' cannot be used on type 'ThisBase1'}}
-    super.baseFunc1(ThisBase1())(42)
+    super.baseFunc1(ThisBase1())(42) // expected-error {{'super' instance member 'baseFunc1' cannot be used on type 'ThisBase1'}}
```

Comment:
@xedin @hborla do you know why the diagnostic output is inconsistent here? It's printing `baseFunc0()` vs `baseFunc1`

Reply:
Hm, I don't know why it mentions `baseFunc0` here when this expression has nothing to do with it...

Reply:
Sorry, I meant compared to the line above, `super.baseFunc0(ThisBase1())()`.

Reply:
Ah, AFAIK that's related to how printer handles names - prints `()` if there are no arguments but avoids printing `(_:)` unless you pass it a special option set...

Reply:
Alright, if this is expected behavior that's fine. I just wanted to make sure I hadn't done something wrong because names got a bit more complicated since the last time I looked at diagnostics :)

Reply:
Yeah, I think that's expected :)